src/lucky/renderable.cr

```diff
@@ -43,7 +43,7 @@ module Lucky::Renderable
   #   end
   # end
   # ```
-  macro render(page_class = nil, **assigns)
```

Comment:
Should we have this anyway to just raise an exception to say it's been renamed?

Reply:
Good point :)